src/main/java/org/broadinstitute/hellbender/tools/walkers/bqsr/BaseRecalibrator.java

```diff
@@ -176,10 +180,10 @@ public void onTraversalStart() {
         if (RAC.knownSites.isEmpty() && !RAC.RUN_WITHOUT_DBSNP) // Warn the user if no dbSNP file or other variant mask was specified
             throw new UserException.CommandLineException(NO_DBSNP_EXCEPTION);
 
-        requestedCovariates = getCovariatesArray();
+        covariates = new StandardCovariateList();
 
         logger.info("The covariates being used here: ");
-        for (Covariate cov : requestedCovariates) { // list all the covariates being used
+        for (Covariate cov : covariates) { // list all the covariates being used
             logger.info("\t" + cov.getClass().getSimpleName());
             cov.initialize(RAC); // initialize any covariate member variables using the shared argument collection
```

Comment:
Here the covariates are initialized individually by BQSR rather than via the new `StandardCovariateList.initializeAll()`. Either make initialization part of the construction of a `StandardCovariateList` (best option), or delete `StandardCovariateList.initializeAll()`.